src/app/CASESessionManager.h

```diff
@@ -71,7 +71,7 @@ class CASESessionManager
     void FindOrEstablishSession(const ScopedNodeId & peerId, Callback::Callback<OnDeviceConnected> * onConnection,
                                 Callback::Callback<OnDeviceConnectionFailure> * onFailure);
 
-    OperationalDeviceProxy * FindExistingSession(const ScopedNodeId & peerId) const;
+    OperationalSessionSetup * FindExistingSessionSetup(const ScopedNodeId & peerId) const;
 
     void ReleaseSession(const ScopedNodeId & peerId);
```

Comment:
This will get removed in https://github.com/project-chip/connectedhomeip/issues/20452, but we will still need `void ReleaseSession(OperationalSessionSetup * device) ` after that issue is addressed